cache/cache.go

```diff
@@ -89,15 +90,16 @@ func Create(config config.Config, inFile []string, outFile []string, manifestfil
         if filepath.IsAbs(ofile){
             continue
         }
-        fullPath := filepath.Join(dirOfCachedOutputFiles, filepath.Dir(ofile))
-        if !utils.Exists(fullPath){
-            err = os.MkdirAll(fullPath, 0775)
-            if err != nil{
-                return err
-            }
+        target := filepath.Join(dirOfCachedOutputFiles, strings.Replace(ofile, "/", ".", -1))
+        cpCmd := exec.Command("cp", filepath.Join(config.BaseDir, ofile), target)
+        err = cpCmd.Run()
+        if err != nil{
+            return err
         }
-        cpCmd := exec.Command("cp", filepath.Join(config.BaseDir, ofile),
-                              filepath.Join(fullPath, filepath.Base(ofile)))
+    }
```

Comment:
We might want to use goroutines to parallelize the copy/symlink operations. This commit is fine. But in a subsequent commit, we should do that.

Reply:
took care of them in the new commit